en/editor/gameview/index.md

```diff
@@ -0,0 +1,35 @@
+## Game view description
+
+![image](./gameview.png)
+
+`GameView` is a game view function embedded in the editor. It can run the game in the editor without opening the browser. The advantage is that it can adjust the model and state of the game in real time through gizmo or other plugins of the editor, etc,achieving "what you see is what you get" when the game is running.
```

Comment:
editor, etc,achieving "what you see is what you get" when the game is running.
-->
editor, etc, achieving __what you see, is what you get__ when the game is running.